app/controllers/admin/edition_tags_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::EditionTagsController < Admin::BaseController
+  before_filter :find_edition
+  before_filter :enforce_permissions!
+  before_filter :limit_edition_access!
+
+  def enforce_permissions!
```

Comment:
Will change it, also will do another PR to make all of them private methods (seems to be a mix between public/private in different controllers)